tensorflow/core/kernels/mkl/mkl_matmul_ops_common.h

```diff
@@ -33,8 +33,17 @@ using mkldnn::stream;
 
 namespace tensorflow {
 
+#define L1_SIZE 32 * 1024
```

Comment:
Just FYI, TensorFlow can read cache sizes through Eigen, although that may come with some latency costs.
https://github.com/tensorflow/tensorflow/blob/add0a785e5af7d39cbb6801e2aefd83db39738b0/tensorflow/core/kernels/conv_grad_ops_3d.cc#L395